docs/streams/stream-basics/consumers.md

```diff
@@ -38,3 +38,60 @@ Client libraries can provide their own listener implementations for consumers. I
 ## Cursor
 
 A cursor is the subscription position for a stream consumer. You can think of it like the cursor on your page, marking where you are.
+
+## Create Consumers
+
+It is possible to use our JavaScript or Python SDK to create producers.
+
+<Tabs groupId="modify-single">
+<TabItem value="javascript" label=" JavaScript SDK">
+
+- Step 1. [Install the SDK](../../sdks/install-sdks.md).
+- Step 2. Change `BASE_URL` if necessary and insert `apiKey`.
+- Step 3. Create an instance of the jsC8.
+- Step 4. Request `stream` object.
+- Step 5. Request One Time Password and create consumer.
+
+```js
+const jsc8 = require("jsc8");
+
+const BASE_URL = "https://play.paas.macrometa.io/"
+
+client = new jsc8({
+    url: BASE_URL,
+    apiKey: "xxxxxx",
+    fabricName: "_system",
+});
+
+const streamName = "streamQuickstart";
+const subscriptionName = "consumer-subscription"
+
+async function consumer() {
+  try {
```

Comment:
@dlozina-macrometa we should add the createStream block here too

Reply:
Also Whatever changes we make here is automatically applicable to all examples in subscriptions folder since we are using the same consumer code there

Reply:
Do we actually need it here? I have imagined that people that are testing with code examples will first run `producer.js` code, we created stream there. @koshy-thomas

Reply:
Yeah I guess its fine then